projects/ad2s1210_iio/app/ad2s1210_iio.c

```diff
@@ -0,0 +1,619 @@
+/***************************************************************************//**
+ *   @file    ad2s1210_iio.c
+ *   @brief   Implementation of AD2S1210 IIO application interfaces
+********************************************************************************
+ * Copyright (c) 2023 Analog Devices, Inc.
+ * Copyright (c) 2023 BayLibre, SAS.
+ * All rights reserved.
+ *
+ * This software is proprietary to Analog Devices, Inc. and its licensors.
+ * By using this software you agree to the terms of the associated
+ * Analog Devices Software License Agreement.
+*******************************************************************************/
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+#include "iio.h"
+#include "iio_types.h"
```

Comment:
iio_types.h and iio.h are already included within the iio_trigger. 
Do we need it here?

Reply:
ok ill remove it. (as a side note: i usually would explicitly include the headers that are needed the c files to avoid depending on other h headers to include what is need. )